modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/java/com/liferay/frontend/taglib/clay/sample/web/internal/portlet/ClaySamplePortlet.java

```diff
@@ -70,6 +73,10 @@ public void doDispatch(
 		renderRequest.setAttribute(
 			ClaySamplePortletKeys.NAVIGATION_BARS_DISPLAY_CONTEXT,
 			new NavigationBarsDisplayContext());
+		renderRequest.setAttribute(
```

Comment:
same here, the `frontend-taglib-clay-sample-web` should not receive any modification in this PR

Reply:
Done!